packages/x-charts-pro/src/internals/plugins/useChartProZoom/initializeZoomInteractionConfig.ts

```diff
@@ -86,6 +89,10 @@ function initializeFor<T extends 'zoom' | 'pan'>(
           }
         : {},
     };
+
+    if (type === 'wheel' && interactionType === 'pan') {
+      acc[type].allowedDirection = lastEmpty?.allowedDirection ?? 'x';
+    }
```

Comment:
Why do we need this special case?

Reply:
We don't specifically need this, it is mostly to not add a "allowedDirection='x'" to every object